apps/spark/src/main/java/com/linkedin/openhouse/jobs/util/TableStatsCollectorUtil.java

```diff
@@ -197,6 +201,7 @@ protected static IcebergTableStats populateTableMetadata(Table table, IcebergTab
         .sharingEnabled(
             policyMap.containsKey("sharingEnabled") && (Boolean) policyMap.get("sharingEnabled"))
         .retentionPolicies(buildRetentionStats(policyMap))
+        .earliestPartitionDate(getEarliestPartitionDate(table, spark, policyMap))
```

Comment:
Can we have this as part IcebergTableStats or even RetentionStats? Lets not combine earliestPartition stat with TableMetadata.

Reply:
added stat to IcebergTableStats 👍